modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceVirtualProducts.testcase

```diff
@@ -616,4 +616,124 @@ definition {
 		}
 	}
 
+	@description = "COMMERCE-9794 | As a Admin I want to be able to view the Setting Tab on Order Item from a virtual product"
+	@priority = 4
+	test CanViewVirtualProductSettingsOnOrderItemPage {
+		property portal.acceptance = "false";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a business account with a buyer user are created") {
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And a virtual product is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				gtin = "GTIN1",
+				manufacturerPartNumber = "MPN1",
+				productDescription = "Full Description",
+				productName = "Virtual Product",
+				productType = "Virtual",
+				shortDescription = "Short Description",
+				sku = "VIRTUALSKU");
+		}
+
+		task ("And the admin selects a file for the virtual product along with the activation status") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "Virtual");
+
+			CommerceNavigator.gotoEntry(entryName = "Virtual Product");
+
+			CommerceEntry.gotoMenuTab(menuTab = "Virtual");
+
+			Click(locator1 = "FormFields#SELECT_UPLOAD_FIELD");
+
+			SelectFrameTop();
+
+			SelectFrame(
+				key_frameTitle = "Select File",
+				locator1 = "CommerceEntry#ANY_IFRAME");
+
+			UploadCommonFile.uploadCommonFileHiddenNoMouseOver(
+				locator1 = "TextInput#FILE",
+				value1 = "Commerce_White.jpg");
+
+			Click(
+				key_filterOption = "Add",
+				locator1 = "AppBuilder#CHOOSE_SEARCH_FILTER_BY_OPTION");
+
+			CommerceProducts.updateProductVirtualActivationStatus(
+				key_contentType = "Base Information",
+				key_fieldLabel = "Activation Status",
+				status = "Completed");
```

Comment:
There's already a macro that uploads a file for a virtual product: `CommerceProducts.updateProductVirtualConfiguration()`, please use this one. Adjust the macro to include the activation status change macro(`updateProductVirtualActivationStatus`). Thanks!